index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#" class="logo"> My Logo </a>
+        <div class="menu">
+          <img src="images/svg/Menu.svg" alt="Menu" />
+        </div>
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#" class="hover-blue-text"> Portfolio </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="hover-blue-text"> About </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="hover-blue-text"> Contact </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="header">
+      <div class="container">
```

Comment:
- i kindly suggest that you make use of semantic `section` tag It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.